spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-catalog-watcher/src/test/resources/busybox/deployment.yaml

```diff
@@ -17,6 +17,6 @@ spec:
       containers:
         - name: busybox
           # image: arm64/busybox:latest
-          image: busybox:latest
```

Comment:
`latest` is like a `SNAPSHOT`, not a good plan if we want reliable tests. This is totally my mistake